docs/configurations.md

```diff
@@ -165,59 +196,82 @@ The following configurations are available for JDBC transactions:
+| `scalar.db.server.prometheus_exporter_port`       | Prometheus exporter port. Prometheus exporter will not be started if a negative number is given. | `8080`                  |
+| `scalar.db.server.grpc.max_inbound_message_size`  | The maximum message size allowed to be received.                                                 | The gRPC default value. |
+| `scalar.db.server.grpc.max_inbound_metadata_size` | The maximum size of metadata allowed to be received.                                             | The gRPC default value. |
+| `scalar.db.server.decommissioning_duration_secs`  | The decommissioning duration in seconds.                                                         | `30`                    |
 
 For details about ScalarDB Server, see [ScalarDB Server](scalardb-server.md).
 
-## Other configurations
+## Other ScalarDB configurations
 
-This section explains other configurations.
-
-Other configurations are available for ScalarDB:
+The following are additional configurations available for ScalarDB:
 
 | Name                                                             | Description                                                                                                                                                                                                       | Default              |
 |------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------|
 | `scalar.db.metadata.cache_expiration_time_secs`                  | ScalarDB has a metadata cache to reduce the number of requests to the database. This setting specifies the expiration time of the cache in seconds.                                                               | `-1` (no expiration) |
 | `scalar.db.active_transaction_management.expiration_time_millis` | ScalarDB maintains ongoing transactions, which can be resumed by using a transaction ID. This setting specifies the expiration time of this transaction management feature in milliseconds.                       | `-1` (no expiration) |
-| `scalar.db.default_namespace_name`                               | The given namespace name will be used by operations that do not already specify a namespace. If you would like to use this setting with ScalarDB Server, configure this setting on the client-side configuration. |                      |
+| `scalar.db.default_namespace_name`                               | The given namespace name will be used by operations that do not already specify a namespace. If you would like to use this setting with ScalarDB Server, configure this setting in the client-side configuration. |                      |
```

Comment:
For `master`, `3.10`, and `3.9` branches, can we simply remove the last sentence that mentions `ScalarDB Server` in the Description column?

Reply:
Yes. We can remove the last sentence.

Reply:
OK. Since this PR is specifically for branches `3.8` and earlier (as well as `master`), I'll be sure to remove this sentence from branches `3.9` and `3.10`, and then update `master` not include it.

Reply:
@josh-wong Let me clarify.

The `master` branch is for the next major release `4.0`. For this branch, we should also remove ScalarDB Server things and mention ScalarDB Cluster instead.

So, you would need to create three separate PRs, correct?

1. This PR for branches `3.8` and earlier.
2. Another PR for the same changes, excluding ScalarDB Server things, for branches `3.9` and `3.10`.
3. Lastly, a PR to remove ScalarDB Server things for the `master` branch.

Reply:
@brfrn169 I think we can have just one PR for steps 2 and 3 as follows:

- Branches for this PR, which mentions ScalarDB Server:
  - `master` (current branch requesting to be merged into in this PR)
  - `3.8` to `3.5` 
- Branches in the follow-up PR, which would mention ScalarDB Cluster instead of ScalarDB Server:
  - `master` (update to the branch that this current PR )
  - `3.10` and `3.9`

Please let me know if I'm misunderstanding.

Reply:
I understand, but it looks like we don’t need to apply the changes of this PR to `master`.

Thus, it seems suitable to merge this PR into branches `3.5` to `3.8`, and subsequently, we can create a follow-up PR for `master`, `3.9`, and `3.10`.

Reply:
OK, no worries then. I'll change the target branch to `3.8`. 

(I actually didn't realize I could change the target branch for this PR on my own...)